setup.py

```diff
@@ -173,9 +173,33 @@ class MyInstallData (install_data, object):
 
     def run (self):
         """Adjust permissions on POSIX systems."""
+        self.install_translations()
         super(MyInstallData, self).run()
         self.fix_permissions()
 
+    def install_translations (self):
+        """Install compiled gettext catalogs."""
+        # A hack to fix https://github.com/linkcheck/linkchecker/issues/102
+        i18n_files = [
+            (dir, files) for (dir, files) in self.data_files
+            if 'LC_MESSAGES' in dir
+        ]
+        self.data_files = [
+            (dir, files) for (dir, files) in self.data_files
+            if 'LC_MESSAGES' not in dir
+        ]
```

Comment:
Copying comment over from https://github.com/linkcheck/linkchecker/commit/d94bd0ee0660a989eb49d4179ed1db7af9c766d9#commitcomment-25354512:
> at the very least we should remove duplication between those two initializers which are identical.

The only other clean-ish way to split the list into two that I can come up with is
```python
i18n_files = []
data_files = []
for dir, files in self.data_files:
    if 'LC_MESSAGES' in dir:
        i18n_files.append((dir, files))
    else:
        data_files.append((dir, files))
self.data_files = data_files
```

and I'm not sure it's an improvement.  What do you think?

Reply:
that sure looks better. i see now why it's two different loops. it does seem strange to recreate the `data_files` list here - it seems to me it would be better to generate it properly from the start...